creator/Documents/ScoreboardIntroduction.md

```diff
@@ -82,62 +82,62 @@ This command returns a list of all of the objectives in the world.
 To remove an objective from the game use:
 
 ```
-/scoreboard objective remove [objective name]
+/scoreboard objective remove <objective: string>
 ```
 
-### setdisplay list
+### setdisplay 
 
-Use the **list** option of **setdisplay** to see scoreboard information listed on the pause screen. You can specify whether the players are ranked in ascending or descending order, according to score.
+You can modify how scoreboard objectives are displayed in the game. There are three slots that an objective can be displayed in: **belowName**, **list**, and **sidebar**. Specifying a slot without an objective name clears that display slot.
+
+Use the **list** option to see scoreboard information listed on the pause screen. You can specify whether the players are ranked in ascending or descending order, according to score.
 
 ```
-/scoreboard objectives setdisplay list [objective] [ascending|descending]
+/scoreboard objectives setdisplay list [objective: string] [ascending|descending]
 ```
 
  ![Image of player names displayed in a list on pause screen](Media/Commands/scoreboard_setdisplay_list.png)
 
-### setdisplay sidebar
-
 Use the **sidebar** option to display objective information on the main screen. You can specify whether the players are ranked in ascending or descending order, according to score.
-
-### setdisplay belowname
-
-Use **setdisplay belowname [objective]** to display objective information below player names.
-
 ```
-/scoreboard objectives setdisplay belowname <objective> 
+/scoreboard objectives setdisplay sidebar [objective: string] [ascending|descending]
 ```
 
-If you use the command without specifying an objective, the "belowname" slot will be cleared.
+Use the **belowname** option to display objective information below player names in the world.
 
+```
+/scoreboard objectives setdisplay belowname [objective: string]
+```
 
 ## Scoreboard Players Commands
 
-### add
+The following scoreboard player commands affect a player's score value. The arguments can be listed using the chat’s auto-complete feature.
 
-You can use this to add players to an existing objective. The players don't have to be active in the world at the time or even be real players at all. The objective must already exist and you have to give them a score or you get an error.  
+### set
```

Comment:
I moved the **set** argument first. I felt this was the most important to mention first, since it's the most fundamental operation and it was mentioned earlier in the article.